3.10/http/document-address-and-etag.md

```diff
@@ -163,3 +163,87 @@ unchanged, a *HTTP 412* (precondition failed) error is returned.
 If you want to query, replace, update or delete a document, then you
 can use the *If-Match* header. If the document has changed, then the
 operation is aborted and an *HTTP 412* error is returned.
+
+Dirty Reads
```

Comment:
It's titled "Dirty Reads" here, but "Read from followers" in the Release Notes. If I understand correctly, a dirty read is if the user actually observes a data inconsistency because of a read from a shard follower. Should we thus call the feature "Read from Followers" and use the term "dirty read" only to refer to the side effects?

Reply:
This is politics. The official name of the feature is "Read from followers". The heading "Dirty reads" was a try to point the user towards the potential problems...